### PR TITLE
Master fix 

### PR DESCRIPTION
**Pull request by JacksonKolb94**
I removed commented line of code in master causing problems (and made a style change for readability)

### CODE REVIEW

File: onthequad/onthequad/EventEndPoints.py [hunk 1]
```diff
@@ -26,8 +26,8 @@ def on_get(self, req, resp):
         elif 'myevents' in params:
             send = EventDatabase.getMyEvents(params['myevents'])
         else:
-            # send = EventDatabase.getFutureEvents()
-            send = EventDatabase.getTest("-LaSLgTAWSqB-DUQHoZA")
+            send = EventDatabase.getFutureEvents()
+            # send = EventDatabase.getTest("-LaSLgTAWSqB-DUQHoZA")
```
Comment (paugajt) [inline onthequad/onthequad/EventEndPoints.py]: We should remove commented out code

File: onthequad/onthequad/EventEndPoints.py [hunk 2]
```diff
@@ -39,9 +39,17 @@ def on_post(self, req, resp):
...
+                                      data['endTime'],
+                                      data['category'],
+                                      data['creator'],
+                                      data['availableSpots'],
+                                      data['description'],
+                                      data['coordinates'],
+                                      data['public'])
                 except:
                     send = {"msg": "invalid fields"}
                     resp.body = json.dumps(send)
```
Comment (paugajt) [inline onthequad/onthequad/EventEndPoints.py]: We need to set a response status here

Comment (JacksonKolb94) [reply onthequad/onthequad/EventEndPoints.py]: It looks to me like we are setting the status in every spot.

Comment (paugajt) [reply onthequad/onthequad/EventEndPoints.py]: The exception block doesn't have a `resp.status = falcon.HTTP_<status>`. At least that's what it looks like to me

Comment (JacksonKolb94) [reply onthequad/onthequad/EventEndPoints.py]: Screen Shot 2019-04-10 at 3 29 58 PM

Comment (JacksonKolb94) [reply onthequad/onthequad/EventEndPoints.py]: see at the bottom? @paugajt

**Pull request merged**